docs/Installation.md

```diff
@@ -731,63 +700,71 @@ service postgresql initdb
 service postgresql start
 ```
 
-Initialize Zonemaster database and user (unless you keep an old database):
+To create the database and the database user (unless you keep an old database)
+run the command on one line.
 
 ```sh
-sudo -u postgres createuser -P zonemaster
+psql -U postgres -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';"  -c "CREATE DATABASE zonemaster WITH ENCODING 'UTF8';"

```

Comment:
same remark and suggestion as with CentOS

Reply:
Here is no question on sudo and no prompting so two commands should work.